bsp/esp-box-3/README.md

```diff
@@ -12,3 +12,16 @@ ESP32-S3-BOX-3 comes with 16MB of QSPI flash and 16MB of Octal PSRAM. ESP32-S3-B
 
 ESP32-S3-BOX-3 also uses a Type-C USB connector that provides 5 V of power input, while also supporting serial and JTAG debugging, as well as a programming interface; all through the same connector.
 
+<!-- Autogenerated start: Dependencies -->
+### Dependencies
+|           component           |version|
+|-------------------------------|-------|
+|        espressif/button       | >=2.5 |
```

Comment:
Is it possible add the link to each component. I think, there can be added link like this "https://components.espressif.com/components/" + "espressif/button"

Example:
|           component           |version|
|-------------------------------|-------|
|        [espressif/button](https://components.espressif.com/components/espressif/button)       | >=2.5 |

Reply:
Thank you for the review @espzav ! updated!